interfaces/HO-Portal/src/app/pages/payment/payment.page.ts

```diff
@@ -10,18 +12,33 @@ import { ProgramPhase } from 'src/app/models/program.model';
 })
 export class PaymentPage implements OnInit {
   public programId = this.route.snapshot.params.id;
+  public program: Program;
   public thisPhase = ProgramPhase.payment;
   public isReady: boolean;
 
   public userRole = this.authService.getUserRole();
 
+  public enumExportType = ExportType;
+
   constructor(
     private route: ActivatedRoute,
     private authService: AuthService,
+    private programsService: ProgramsServiceApiService,
   ) {}
 
   ngOnInit() {}
 
+  async isIntersolve() {
+    const intersolveString = 'intersolve';
```

Comment:
I think it's not really useful to have this very verbose extra string here; as it isn't reused anywhere anyway..